rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/dataspace/RestDataspaceImpl.java

```diff
@@ -400,20 +420,26 @@ private Response serverErrorRes(String format, Object... args) {
                 (args == null || args.length == 0) ? format : String.format(format, args)).build();
     }
 
-    private FileObject resolveFile(Session session, String dataspace, String pathname)
-            throws FileSystemException, NotConnectedException, PermissionException {
-        return "user".equals(dataspace) ? fileSystem(session).resolveFileInUserspace(pathname) : fileSystem(
-                session).resolveFileInGlobalspace(pathname);
+    public FileObject resolveFile(Session session, String dataspace, String pathname)
+            throws FileSystemException, NotConnectedRestException, PermissionRestException {
+        try {
+            return USER.equalsIgnoreCase(dataspace) | SchedulerConstants.USERSPACE_NAME.toString().equalsIgnoreCase(dataspace) ? fileSystem(session).resolveFileInUserspace(pathname) : fileSystem(
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Correct this "|" to "||". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS2178)